pep-3145.txt

```diff
@@ -2,7 +2,7 @@ PEP: 3145
 Title: Asynchronous I/O For subprocess.Popen
 Version: $Revision$
 Last-Modified: $Date$
-Author: (James) Eric Pruitt, Charles R. McCreary, Josiah Carlson
```

Comment:
I don't think we should change people's names without consultation.

Reply:
I didn't want to change this, but unfortunately as `()` is the key element of the legacy authors email syntax, this breaks parsing and validation (at least without a bunch of hacks and/or special cases in the checkers and every tool that parses it). I meant to ping @ericpruitt here, but unfortunately it slipped my mind due to how late (≈2 am local) I finally finished this.

@ericpruitt , how do you want your name to be written here, preferably without parenthesis if that's okay? Just Eric Pruitt, as you list on your GitHub profile (as it currently is updated to)? Or J. Eric Pruitt, James Eric Pruitt or something else?

Reply:
Please write it as just "Eric Pruitt".

Reply:
Thanks, that's what it's updated it to now :+1: